topology_test.py

```diff
@@ -476,6 +478,8 @@ def test_stop_decommission_too_few_replicas_multi_dc(self):
         @jira_ticket CASSANDRA-12510
         @expected_errors ToolError when # nodes will drop below configured replicas in NTS/SimpleStrategy
         """
+        self.fixture_dtest_setup.ignore_log_patterns = (r'.*Not enough live nodes to maintain replication factor*')
```

Comment:
We need to ignore this in logs, it is a legitimate log to skip as we are testing this scenario but without it, the node would stop suddenly because internal scanning of the logs by dtest framework would falsely evaluated that node to be eligible for termination.

Reply:
Comments around this would be useful, it's not obvious what we're doing here.